lib/data_element.rb

```diff
@@ -12,23 +12,24 @@ class << self
       def create(elements)
         elements = [elements].flatten
         category_combo_id = JSON.parse(Dhis2.resource["categoryCombos"].get)["categoryCombos"]
-                            .first["id"]
+                            .find{ |categoryCombo| categoryCombo["displayName"] == 'default'}["id"]
```

Comment:
You are getting all the combos, then filtering the array in Ruby - no option to put a filter on the DHIS2 request instead?